templates/yeet/Makefile

```diff
@@ -0,0 +1,89 @@
+.PHONY: default \
+debug \
+pkg \
+unpkg \
+clean \
+vmlinux \
+clean_vmlinux \
+format \
+check_format \
+container
+
+PROJECT_ROOT=$(shell git rev-parse --show-toplevel)
+
+BINDIR = bin
+TARGET = __TARGET__
+SRCDIR = src
+BUILDDIR = build
+VMLINUX ?= /sys/kernel/btf/vmlinux
+CFORMAT = .clang-format
+
+CC = clang-19
+CFLAGS = -g -O2 -target bpf -I include
+
+SRCS := $(wildcard $(SRCDIR)/*.bpf.c)
+OBJS = $(patsubst $(SRCDIR)/%.c,$(BUILDDIR)/%.o,$(SRCS))
+
+CONTAINER ?= $(shell test -e /.dockerenv && echo no || echo yes)
+
+default: $(if $(CONTAINER), container,vmlinux $(BUILDDIR) $(BINDIR) $(BINDIR)/$(TARGET))
+ifeq ($(CONTAINER), yes)
+	docker run --rm -it -v .:/opt/$(TARGET) -w /opt/$(TARGET) $(TARGET) make
+endif
+
+debug: CFLAGS += -DBPF_DEBUG=1
+debug: default
+
+pkg:
+	$(PROJECT_ROOT)/scripts/yeet_pkg.sh --target $(TARGET) --bin $(BINDIR)/$(TARGET)
+
+unpkg:
+	$(PROJECT_ROOT)/scripts/yeet_pkg.sh -u --target $(TARGET)
+
+container:
+	docker build -t $(TARGET) .
+
+$(BINDIR)/$(TARGET): $(OBJS)
+	bpftool gen object $@ $^
+	chmod +x $@
+
+$(BINDIR):
+	mkdir -p $(BINDIR)
+
+$(BUILDDIR):
+	mkdir -p $(BUILDDIR)
+
+$(BUILDDIR)/%.bpf.o: $(SRCDIR)/%.bpf.c
+	$(CC) $(CFLAGS) -c $< -o $@
+
+clean:
+	rm -rf $(BUILDDIR)
+	rm -rf $(BINDIR)
+	rm -rf $(TARGET).yeet
+	rm -rf $(TARGET)
+
+vmlinux: $(if $(filter yes,$(CONTAINER)), container, )
```

Comment:
sp after , what about that third arg

Reply:
intentionally empty. Don't need a dependency when $CONTAINER = no